packages/definitions/src/integration.ts

```diff
@@ -49,6 +49,12 @@ export const integrationDefs = {
     iconUrl: "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons@master/svg/qbittorrent.svg",
     category: ["downloadClient", "torrent"],
   },
+  aria2: {
+    name: "Aria2",
+    secretKinds: [[], ["apiKey"]],
```

Comment:
```suggestion
    secretKinds: [["apiKey"], []],
```

I suggest we put the api key option first as we want to encourage people to use auth

Reply:
As discussed we leave it like that for now

Reply:
As discussed, by default there is no API key and I couldn't figure out where to obtain it. Let's keep it like this.